vips/vip-500/bsctestnet.ts

```diff
@@ -0,0 +1,281 @@
+export const SWAP_ROUTER = "0x98C108285E389A5a12EdDa585b70b30C786AFb86";
+
+const { POOL_REGISTRY, VTREASURY, REDSTONE_ORACLE, RESILIENT_ORACLE } = NETWORK_ADDRESSES["bsctestnet"];
+
+export const SOLVBTC_BBN_ONEJUMP_REDSTONE_ORACLE = "0x9767348B2e36E649A42b1DEA975D2474F72C6B96";
+export const PT_SOLVBTC_BBN_PENDLE_ORACLE = "0x31B78f0Ef6A87cF0E812197226306954a65907d5";
+export const MOCK_PENDLE_PT_ORACLE = "0xa37A9127C302fEc17d456a6E1a5643a18a1779aD";
+
+export const PRIME = "0xe840F8EC2Dc50E7D22e5e2991975b9F6e34b62Ad";
+
+export const USDT = "0xA11c8D9DC9b66E209Ef60F0C8D969D3CD988782c";
+export const USDC = "0x16227D60f7a0e586C66B005219dfc887D13C9531";
+export const BTCB = "0xA808e341e8e723DC6BA0Bb5204Bafc2330d7B8e4";
+export const ETH = "0x98f7A83361F7Ac8765CcEBAB1425da6b341958a7";
+export const XVS = "0xB9e0E753630434d7863528cc73CB7AC638a7c8ff";
+export const RISK_FUND_CONVERTER = "0x32Fbf7bBbd79355B86741E3181ef8c1D9bD309Bb";
+export const USDT_PRIME_CONVERTER = "0xf1FA230D25fC5D6CAfe87C5A6F9e1B17Bc6F194E";
+export const USDC_PRIME_CONVERTER = "0x2ecEdE6989d8646c992344fF6C97c72a3f811A13";
+export const BTCB_PRIME_CONVERTER = "0x989A1993C023a45DA141928921C0dE8fD123b7d1";
+export const ETH_PRIME_CONVERTER = "0xf358650A007aa12ecC8dac08CF8929Be7f72A4D9";
+export const XVS_VAULT_CONVERTER = "0x258f49254C758a0E37DAb148ADDAEA851F4b02a2";
+
+const commonSpec = {
+  decimals: 8,
+  exchangeRate: parseUnits("1", 28),
+  comptroller: COMPTROLLER,
+};
+
+export const tokens = {
+  "PT-SolvBTC.BBN-27MAR2025": {
+    address: "0x964Ea3dC70Ee5b35Ea881cf8416B7a5F50E13f56",
+    decimals: 18,
+    symbol: "PT-SolvBTC.BBN-27MAR2025",
+  },
+  "SolvBTC.BBN": {
+    address: "0x8FD14481C1616d9AdA7195Be60f9d8d0994b9AE1",
+    decimals: 18,
+    symbol: "SolvBTC.BBN",
+  },
+  BTCB: {
+    address: "0xA808e341e8e723DC6BA0Bb5204Bafc2330d7B8e4",
+    decimals: 18,
+    symbol: "BTCB",
+  },
+};
+
+export const newMarkets = {
+  "vPT-SolvBTC.BBN-27MAR2025_BTC": {
+    vToken: {
+      address: "0xf3bF150A7D3d42E8C712e2461102593Dc50266Bb",
+      name: "Venus PT-SolvBTC.BBN-27MAR2025 (BTC)",
+      symbol: "vPT-SolvBTC.BBN-27MAR2025_BTC",
+      underlying: tokens["PT-SolvBTC.BBN-27MAR2025"],
+      ...commonSpec,
+    },
+    riskParameters: {
+      collateralFactor: parseUnits("0.8", 18),
+      liquidationThreshold: parseUnits("0.85", 18),
+      supplyCap: parseUnits("80", 18),
+      borrowCap: parseUnits("0", 18),
+      reserveFactor: parseUnits("0", 18),
+      protocolSeizeShare: parseUnits("0.015", 18),
+    },
+    initialSupply: {
+      amount: parseUnits("0.110483117280995732", 18),
+      vTokenReceiver: VTREASURY,
+    },
+    interestRateModel: {
+      address: "0x87BC48C851B47e199288139e05AB9B9B761cE4DB",
+      base: "0",
+      multiplier: "0.0875",
+      jump: "2.0",
+      kink: "0.8",
+    },
+  },
+  vBTCB_BTC: {
+    vToken: {
+      address: "0x92766B8890E93907FE99B98D655898713053e79e",
```

Comment:
```suggestion
      address: "0x138500e8502f32f004F79507143E4FaaCA03E26d",
```

According to [this](https://github.com/VenusProtocol/isolated-pools/pull/458/files#diff-14ad40eacc338292d4976a675c61cc5fe26bc05747a992effd535dbd147fdcfaR116)

Reply:
https://github.com/VenusProtocol/vips/pull/442/commits/8383638bc7388cf9d15ac16dbc71a8d95ff0e0ce